data/strings/types_strings.txt

```diff
@@ -22655,6 +23255,46 @@
     zh-Hans = 箱包店
     zh-Hant = 箱包店
 
+  [type.shop.bed]
+    en = Beds Shop
+    ar = متجر الأسرة
+    be = Крама ложкаў
+    bg = Магазин за легла
+    ca = Botiga de llits
+    cs = Prodejna postelí
+    da = Senge butik
+    de = Betten Shop
```

Comment:
"Bettengeschäft", also "Matratzengeschäft" should be added as alternative.

Reply:
Yes, no one says "Betten Shop"